src/adapter/z-stack/adapter/backup.ts

```diff
@@ -72,6 +72,12 @@ const items = {
         osal: true,
         product: PRODUCT_ALL,
     },
+    ZCD_NV_TCLK_SEED: {
```

Comment:
Last question; is it actually needed to backup this? For Zstack 3.x.0 we never made a backup of `0x0101` and we did not encounter any issues because of this.

Reply:
I'm not sure, but in all previous releases you backed that up because of the wrong `LEGACY_TCLK_TABLE_START` ID.

@0x3ec, how do you think?

Reply:
We never backed it up for ZStack 3.x.0, I expect the same for ZStack 3.0.x.

Reply:
2592 <-> 2538 migration works fine together with this PR. My collegue @0x3EC helped me with this commit. He is more familar in Z-Stack. So let's wait his answer.

In any case, I can try to migrate 2538 <-> 2562 without backing up `LEGACY_TCLK_TABLE_START`, but I'm not sure that it will be stable.

Reply:
`LEGACY_TCLK_TABLE_START` should be backed up and restored for sure (I'm not sure how this would work out yet because Zstack 3.x.0 doesnt have this). But I think `ZCD_NV_TCLK_SEED` is not needed.

Reply:
@Koenkk , actually you never made backup of the `LEGACY_TCLK_TABLE_START=273` and it was also worked. But I don'n know how :)

Reply:
`ZCD_NV_TCLK_SEED` was generated by the random. It used during genrating link key.
There is no official inforamtion from the TI about migration between stack revisions and this parameter.
We found some info in the TI forums about that, but can't confirm: https://e2e.ti.com/support/wireless-connectivity/zigbee-and-thread/f/158/t/814208

Reply:
The TCLK table contains the Zigbee 3.0 device security key of each device. I however don't know what happens when this is not restored :smile: (but doesn't look like a big deal?).

Reply:
I tried to migrate 2538 -> 2562 without TCLK table and failed.
May be I had not enough patience to wait that ZigBee networked will be restored, but after 5-10mins that did not happned.
So,I think we have enough disk space on the user devices to backup extra 100-300 bytes :)
If you prefer to delete this table from the backup - I can do that. Let me know your desicion.

Reply:
About TCLK table: https://www.ti.com/lit/an/swra671/swra671.pdf?ts=1602185076595&ref_url=https%253A%252F%252Fwww.google.com%252F page 4.

Reply:
The **ZCD_NV_TCLK_SEED** value is used in the **_ZDSecMgrGenerateKeyFromSeed_** function. The function description is as follows: _"Generate the TC link key for an specific device usign seed and ExtAddr"_.

The **_ZDSecMgrGenerateKeyFromSeed_** function is called when a **ZDO_REQUEST_KEY_IND** request is received. 
I have not encountered such requests (RequestKey). 
Maybe a device manufactured in accordance with all the zigbee 3 safety rules can send this request.

Reply:
@Hacker-CB backing up TLCK_SEED can be removed. My point is that NV item `257` was never backed up for ZStack 3.x.0, so I don't see a reason to start backing it up now. This would also prevent a breaking change on the ZStack 3.x.0 backups.

Reply:
I can remove, but in any case it will be breaking change in the backup.
`LEGACY_TCLK_TABLE_START` now has id 273 and before it had 257

Reply:
I've checked, but now I remember that the IDs of the NV items are stored in backup file. So for an old backup LEGACY_TCLK_TABLE_START will have an ID of 257, which is good. So I think there is no breaking change.

Still I think we should remove this, as we shouldn't backup/restore items which we are not sure we should back up at all.

Reply:
I removed TCLK_SEED from the backup.